src/main/resources/locales/en/delete-court-reference-data-confirmation.json

```diff
@@ -12,5 +12,6 @@
         "locationType": "Location Type",
         "jurisdiction": "Jurisdiction",
         "region": "Region"
-    }
+    },
+    "deleteSubscriptionText": "Click here to delete the all the subscription(s) for "
```

Comment:
Think there is a redundant "the" in the sentence. Should it not be "delete all the subscription(s)..."?

Reply:
fixed